index.html

```diff
@@ -0,0 +1,235 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio webpage</title>
+    
+  <meta http-equiv="refresh" content="30">
+
+</head>
+
+<link rel="stylesheet" href="styles.css">
+<link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+
+<body>
+
+    <!-- Nav bar Section -->
+
+    <div class="nav-bar">
+                <div class="contact-button">
+                    <button>+00993390090 </button>
+                </div>
+      <div class="text">
+           <h1>welcome!</h1>
+           <p>let us help build your dream project</p>
+
+      </div>
+        
+    </div>
+
+
+
+    <!--  Section Two -->
+
+
+
+    <section class="Section-two-section">
+        <div class="section-two">
+            <div class="header">
+                <h1>How can we help you?</h1>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Suspendisse et<br> justo. Praesent mattis commodo augue.</p>
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-object-ungroup"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-code"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
```

Comment:
This line of code opens with a "pr" tag. There is no such thing as a pr tag. Please fix it. 🙏